app/actions.py

```diff
@@ -231,6 +235,21 @@ def bookmarkRemove(self):
       (boolean) Whether any bookmarks were removed.
     """
     upperRow, _, lowerRow, _ = self.startAndEnd()
+    return self.bookmarkRemoveFrom(upperRow, lowerRow)
+
+  def bookmarkRemoveFrom(self, upperRow, lowerRow):
+    """
+    Removes bookmarks in the given line interval.
+
+    Args:
+      upperRow: The upper line limit.
+      lowerRow: The lower line limit.
```

Comment:
Thanks for writing doc strings. I think it may help someone in the future to know if (for each of) these, whether they are inclusive or exclusive of the row(s). (e.g. just putting ", inclusive" or ", exclusive" at the end of each line is enough).

Reply:
Good point. I have to admit that I played with both options.